src/screens/WalletScreen/index.js

```diff
@@ -49,35 +50,61 @@ type Actions = {
    * @desc Function to request wallet list update.
    */
   updateWalletList: () => void,
-}
+};
 
 type State = {
   /**
    * @desc Flag to control Refreshing on Pull to Refresh
    */
   pullToRefreshInProgress: boolean,
-}
+  /**
+   * @desc Flag to control opening of transactions Modal
+   */
+  transactionsVisible: boolean,
```

Comment:
So, do you use this field or this.state.transactionModel.visible? If this one is not needed, please remove it.

Reply:
Done